### PR TITLE
Volume API added to contract

### PR DESCRIPTION
**Pull request by Anonycoders**

### CODE REVIEW

**Comment by s-soroosh** [comment]
@Anonycoders plz update the contract based on changes we had in implementation

**Comment by s-soroosh** [comment]
@Anonycoders

File: api-docs/api-contract.yaml [hunk 1]
```diff
@@ -944,6 +1040,32 @@ definitions:
...
+      mounted_to:
+        type: boolean
+        
+  CreateVolumeModel:
+    type: object
+    description: "A model to represent desired volume to be created"
+    properties:
+      name:
+        type: string
+      storage:
```
Comment (s-soroosh) [inline api-docs/api-contract.yaml]: why isn't it called capacity?

File: api-docs/api-contract.yaml [hunk 2]
```diff
@@ -944,6 +1040,32 @@ definitions:
...
+    
+  VolumeModel:
+    type: object
+    description: "A model to represent entities of a volume"
+    properties:
+      name:
+        type: string
+      status:
+        type: string
+      volume:
```
Comment (s-soroosh) [inline api-docs/api-contract.yaml]: what is this?

Comment (Anonycoders) [reply api-docs/api-contract.yaml]: this is the volume name

Comment (Anonycoders) [reply api-docs/api-contract.yaml]: its k8s name

**Pull request merged**